bindings/python/cntk/logging/progress_print.py

```diff
@@ -398,6 +398,7 @@ def on_write_training_summary(self, samples, updates, summaries, aggregate_loss,
         elapsed_seconds = elapsed_milliseconds / 1000
         speed = _avg(samples, elapsed_seconds)
         avg_loss = _avg(aggregate_loss, samples)
+        self.epochs = summaries
```

Comment:
I suppose this won't work in case of checkpoint/crash/restore, but still better than 0